packages/errors/src/errors.ts

```diff
@@ -0,0 +1,1260 @@
+
+        This is the equivalent to seeing the 'sad face' when Chrome dies.
+
+        This can happen for a number of different reasons:
+
+        - You wrote an endless loop and you must fix your own code
+        - There is a memory leak in Cypress (unlikely but possible)
+        - You are running Docker (there is an easy fix for this: see link below)
+        - You are running lots of tests on a memory intense application
+        - You are running in a memory starved VM environment
+        - There are problems with your GPU / GPU drivers
+        - There are browser bugs in Chromium
+
+        You can learn more including how to fix Docker here:
+
+        https://on.cypress.io/renderer-process-crashed`
+  },
+  AUTOMATION_SERVER_DISCONNECTED: () => {
+    return errTemplate`The automation client disconnected. Cannot continue running tests.`
+  },
+  SUPPORT_FILE_NOT_FOUND: (supportFilePath: string) => {
+    return errTemplate`\
+        Your ${fmt.highlight(`supportFile`)} is missing or invalid: ${fmt.path(supportFilePath)}
+
+        The supportFile must be a .js, .ts, .coffee file or be supported by your preprocessor plugin (if configured).
+
+        Fix your support file, or set supportFile to ${fmt.highlightSecondary(`false`)} if a support file is not necessary for your project.
+
+        If you have just renamed the extension of your supportFile, restart Cypress.
+
+        Learn more at https://on.cypress.io/support-file-missing-or-invalid`
+  },
+  PLUGINS_FILE_ERROR: (pluginsFilePath: string, err: Error) => {
+    return errTemplate`\
+        Your ${fmt.highlight(`pluginsFile`)} is invalid: ${fmt.path(pluginsFilePath)}
+
+        It threw an error when required, check the stack trace below:
+
+        ${fmt.stackTrace(err)}
+      `
+  },
+  PLUGINS_FILE_NOT_FOUND: (pluginsFilePath: string) => {
+    return errTemplate`\
+        Your ${fmt.highlight(`pluginsFile`)} was not found at path: ${fmt.path(pluginsFilePath)}
+
+        Create this file, or set pluginsFile to ${fmt.highlightSecondary(`false`)} if a plugins file is not necessary for your project.
+
+        If you have just renamed the extension of your pluginsFile, restart Cypress.
+      `
+  },
+  PLUGINS_DIDNT_EXPORT_FUNCTION: (pluginsFilePath: string, exported: any) => {
+    const code = errPartial`
+      ${fmt.comment(`// ${pluginsFilePath}`)}
+      module.exports = (on, config) => {
+        ${fmt.comment(`// configure plugins here`)}
+      }`
+
+    return errTemplate`\
+      The ${fmt.highlight(`pluginsFile`)} must export a function with the following signature:
+
+      ${fmt.code(code)}
+
+      Instead it exported:
+
+      ${fmt.meta(JSON.stringify(exported, null, 2))}
+
+      Learn more: https://on.cypress.io/plugins-api
+    `
+  },
+  PLUGINS_FUNCTION_ERROR: (pluginsFilePath: string, err: Error) => {
+    return errTemplate`\
+      The function exported by the ${fmt.highlight(`pluginsFile`)} threw an error: ${fmt.path(pluginsFilePath)}
```

Comment:
The point of this PR is to consistently highlight/surface keywords. Adding parenthesis adds inconsistency (see in your suggestions below - as those don't have them). Additionally, the highlighting in both `stdout` and the GUI highlight those words without needing visual characters like `(...)`.

Last point - I'm trying to avoid long keywords (like the absolute path to files) in the middle of a phrase. It ends up being very distracting and keeping the emphasis of the phrase "threw an error" at the beginning prevents the phrase from being chopped up

Reply:
Just seemed weird to have it say "threw an error: /path/to/file". I agree putting it in the middle of a sentence might not be ideal, but maybe a pattern where we display the path in another way, like:

```
The function exported by the pluginsFile threw an error.

pluginsFile location: /path/to/file

The error:

<error>
```

Reply:
yeah I agree it should probably go on its own line. i fixed up the other suggestions but I don't want to do this since a ton of snapshots have to be updated - I'll update it for 10.x though